tvshows/src/components/features/review/ShowReviewSection/ShowReviewSection.tsx

```diff
@@ -1,31 +1,22 @@
 "use client";
-import { IReview, IReviewList } from "@/typings/review.types";
-import { ReviewList } from "../ReviewList/ReviewList";
+import { IReviewList } from "@/typings/review.types";
 import { ReviewForm } from "../ReviewForm/ReviewForm";
+import { ReviewList } from "../ReviewList/ReviewList";
 
 interface IShowReviewSectionProps {
   showId: number;
   reviewList: IReviewList;
-  addShowReview: (review: IReview) => void;
-  deleteShowReview: (review: IReview) => void;
 }
 
 export const ShowReviewSection = ({
   showId,
   reviewList,
-  addShowReview,
-  deleteShowReview,
 }: IShowReviewSectionProps) => {
-
-  const handleAddReview = (review: IReview) => {
-    review.show_id = showId;
-    addShowReview(review);
-  }
   return (
     <>
-      <ReviewForm onAdd={handleAddReview} />
+      <ReviewForm showId = {showId} />
```

Comment:
Why not fetch the reviews in this component?

Reply:
Ako raspodijelim fetchanje na ShowReviewSection i ShowDetailsCard onda više u ShowContaineru ne mogu provjeriti isLoading i error? Nakon te raspodjele, loading i error se provjeravaju u svakoj komponenti zasebno, ali onda mi se dvaput prikaže poruka Loading...
Što mi je činiti?

Reply:
To ti sada ne izgleda lijepo zato sto ti UI pokaze obicni `Loading` tekst. 
U stvarnom slucaju bi koristila https://v2.chakra-ui.com/docs/components/skeleton ili nesto slicno pa bi to OK izgledalo.
Nemoj se sada brinuti s tim sto se loading prikazuje 2 puta.

Reply:
Okej, pushala sam to. Nisam stigla sad u apartmanu primijenit to i u Homework 6 jer smo u nekoj žurbi bili, ali ako hoćeš, možeš ti to mergati. Jučer sam radila i na popravcima iz Homework 6 tako da bi to trebalo bit to kad se spoji s ovim najnovijim dijelom.

Reply:
Riješeno :)